src/constants/actionTypes.js

```diff
@@ -95,7 +95,14 @@ export const LAYER_EDIT_RELATIONSHIP_LINE_COLOR_SET =
 export const LAYER_EDIT_ORGANISATION_UNITS_SET =
     'LAYER_EDIT_ORGANISATION_UNITS_SET'
 export const LAYER_EDIT_ORGANISATION_UNIT_GROUP_SET =
-    'LAYER_EDIT_ORGANISATION_UNIT_GROUP_SET'
+    'LAYER_EDIT_ORGANISATION_UNIT_GROUP_SET' // Facility TODO: Distinguish frome above
```

Comment:
@turban I don't see these new constants used anywhere. What is the purpose of them?